cmd/controller-manager/app/controller_manager.go

```diff
@@ -111,21 +112,30 @@ func Run(controllerManagerOptions *options.ControllerManagerServer) error {
 		rest.AddUserAgent(k8sKubeconfig, controllerManagerAgentName),
 	)
 	if err != nil {
-		glog.Fatalf("Invalid k8s API configuration: %v", err)
+		return fmt.Errorf("invalid Kubernetes API configuration: %v", err)
```

Comment:
We should end up in main
https://github.com/kubernetes-incubator/service-catalog/blob/master/cmd/controller-manager/main.go#L54

The behavior shouldn't change beyond we actually exit instead of continuing to run.